sklearn/ensemble/tests/test_voting_classifier.py

```diff
@@ -14,11 +16,36 @@
 from sklearn.svm import SVC
 from sklearn.multiclass import OneVsRestClassifier
 
+
 # Load the iris dataset and randomly permute it
 iris = datasets.load_iris()
 X, y = iris.data[:, 1:3], iris.target
 
 
+def test_init_parameters():
+    eclf = VotingClassifier(estimators=[('lr1', LogisticRegression()),
+                                        ('lr2', LogisticRegression())],
+                                        voting='hard')
+    eclf = VotingClassifier(estimators=[])
+    assert_raises(AttributeError, eclf.fit, X, y)
+    assert_raises(AttributeError, eclf.predict_proba, X)
+
+def test_unfitted():
+    """Raise error if VotingClassifier was not fitted"""
+    eclf = VotingClassifier(estimators=[('lr1', LogisticRegression()),
+                                        ('lr2', LogisticRegression())],
+                            voting='soft')
+
+    assert_raises(NotFittedError, eclf.predict, X)
+    assert_raises(NotFittedError, eclf.transform, X)
+    assert_raises(NotFittedError, eclf.predict_proba, X)
+
+    eclf._estimators = []
```

Comment:
Oops sorry . But I am now even more confused. Where is `eclf._estimators` set in the code? There doesn't seem to be such an attribute.